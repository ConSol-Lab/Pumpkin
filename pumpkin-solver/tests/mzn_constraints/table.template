var 1..10: x1 :: output_var;
var 1..10: x2 :: output_var;
var 1..10: x3 :: output_var;

constraint gecode_table_int([x1, x2, x3], [2, 4, 6, 1, 3, 10]);

solve satisfy;
