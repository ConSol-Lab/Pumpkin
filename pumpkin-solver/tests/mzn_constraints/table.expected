x1 = 2;
x2 = 4;
x3 = 6;
----------
x1 = 1;
x2 = 3;
x3 = 10;
----------
==========
